tests/Middleware/CsrfTest.php

```diff
@@ -24,11 +24,33 @@ final class CsrfTest extends TestCase
     /**
      * @test
      */
-    public function validTokenInBodyResultIn200()
+    public function validTokenInBodyPostRequestResultIn200()
     {
-        $token = $this->generateToken();
+        $token      = $this->generateToken();
```

Comment:
Please don't align lines like that, it takes time to align it back :(

Reply:
fix